lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1854,9 +1863,16 @@ void InterpreterFunction::fwdQuantizeInst(const glow::QuantizeInst *I) {
   TensorQuantizationParams params{destTensor->getType().getScale(),
                                   destTensor->getType().getOffset()};
 
-  auto destHandle = destTensor->getHandle<int8_t>();
-  for (size_t i = 0, e = destHandle.size(); i < e; ++i) {
-    destHandle.raw(i) = quantization::quantize(srcHandle.raw(i), params);
+  if (destTensor->getType().getElementType() == ElemKind::Int8QTy) {
```

Comment:
The bias is quantized to int32_t. Since I don't want to create another "Quantize" node whose result is with Int32QTy type, the "Quantize" node is modified to allow its result to be both Int8QTy and Int32QTy.